addon/templates/components/uni-header-menu.hbs

```diff
@@ -0,0 +1,14 @@
+{{#if withExit}}
+    <button class="uni-header__nav__button" {{action "triggerExit"}}>Exit</button>
```

Comment:
Insted of exit, we should have a parameter so we can use translations here and customize the Exit label :)